src/angular-local-storage.js

```diff
@@ -162,7 +162,12 @@ angularLocalStorage.provider('localStorageService', function() {
       // angular.toJson will convert null to 'null', so a proper conversion is needed
       // FIXME not a perfect solution, since a valid 'null' string can't be stored
       if (!item || item === 'null') {
-        return null;
+        if(defaultValue !== null && defaultValue !== undefined){
```

Comment:
Can you please use `isUndefined` instead ?
(and also add spaces. e.g: `if (expression) { ... }`)